plugin-flex-ts-template-v2/src/feature-library/keyboard-shortcuts/utils/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const throttleSettingExplanation = `The throttle time in milliseconds which can be set to prevent the shortcut from being triggered again within the set amount of time. Default value is 50ms to prevent Flex from being overloaded by repeated firings, and to provide a more seamless visual experience.`;
+
+export const deleteSettingExplanation = `This setting enables the Agent to delete individual keyboard shortcut mappings.`;
```

Comment:
The setting applies to the current user is my understanding, so maybe rather than saying "the agent" it can say "you"?

Reply:
Wording updated!